src/MobileDevice/widget/MobileDevice.ts

```diff
@@ -135,11 +149,18 @@ class MobileDevice extends WidgetBase {
                     window.mx.ui.error(`Error while executing microflow ${this.microflow}: ${error.message}`),
                 origin: this.mxform
             });
+        } else if (this.onDeviceReadyAction === "callNanoflow" && this.nanoflow.nanoflow) {
+            window.mx.data.callNanoflow({
+                context,
+                error: error => mx.ui.error(`Error while executing nanoflow ${error.message}`),
+                nanoflow: this.nanoflow,
+                origin: this.mxform
+            });
```

Comment:
order of the arguments should be in the order of inporance... or the order of the API documentation
```ts
nanoflow: this.nanoflow,
context,
origin: this.mxform,
error: error => mx.ui.error(`Error while executing nanoflow ${error.message}`)
````